### PR TITLE
Fix extracting of zips and getDownloadLink for macosx. + New version of OC binaries.

### PR DESCRIPTION
**Pull request by psrna**

### CODE REVIEW

**Comment by psrna** [comment]
There are couple of things in this PR: 

- OC 1.3.1 links are giving me 404, so I replaced it with new version of OC tools - 1.3.2.
- Platform.OS_MACOSX.equals(Platform.getOSArch()) is never true, getOSArch() returns "x86_64" and Platform.OS_MACOSX == "macosx"
- tar.gz archive contains directory with oc tools, in zip archives (both mac + win) there is no directory. Fixed with creating the extractedDirectory - so that I do not need to change the code which creates symlinks etc.

@rhopp @adietish can you please have a look and review. thanks!

**Comment by psrna** [comment]
testPR

File: plugins/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/requirement/OpenShiftCommandLineToolsRequirement.java [hunk 1]
```diff
@@ -139,7 +139,8 @@ private String extractArchive(String fileName, File outputDirectory) {
 		String extractedDirectory = null;
 		if (fileName.endsWith(SUFFIX_ZIP)) {
 			extractedDirectory = StringUtils.chomp(fileName, SUFFIX_ZIP);
-			FileHelper.unzipFile(new File(fileName), outputDirectory);
+			FileHelper.createDirectory(new File(extractedDirectory));
```
Comment (adietish) [inline plugins/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/requirement/OpenShiftCommandLineToolsRequirement.java]: looking at the source of FileHelper, your change seems required. I believe that this should be fixed in FileHelper though, making sure unzip and un-targz behave the same - creating the destination directory if it doesnt.

Comment (psrna) [inline plugins/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/requirement/OpenShiftCommandLineToolsRequirement.java]: Ok, I'll do that. thanks!



But I can

**Comment by psrna** [comment]
@rhopp @adietish ready to review again ;)

**Pull request merged**